packages/loader/container-loader/src/container.ts

```diff
@@ -633,7 +633,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
             // Propagate current connection state through the system.
             this.propagateConnectionState();
-            this.resumeInternal({ fetchOpsFromStorage: false, reason: "createDetached" });
+            if (!this.closed) {
+                this.resumeInternal({ fetchOpsFromStorage: false, reason: "createDetached" });
```

Comment:
resumeInternal already throws if container is closed. If we go with this change, I think we need to convert check in resumeInternal to an assertion to signify it's pre-condition.

That all said, when / how we get here with closed container, and should resumeInternal() be simply a no-op if called in closed state, i.e. this kind of check be in central place, instead of adding it to every caller?

Reply:
The error that FFX reported feels like app may have closed the container when this is going on as resume internal is called async. So they are having this error after container close. Yes it is a no-op. But should we prevent it from telemetry in that case?
@ruchika-m @vladsud

Reply:
Added assert.